pdokservicesplugin/ui_pdokservicesplugindialog.ui

```diff
@@ -1,406 +1,367 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <ui version="4.0">
-  <class>PdokServicesPlugin</class>
-  <widget class="QDialog" name="PdokServicesPlugin">
-    <property name="geometry">
-      <rect>
-        <x>0</x>
-        <y>0</y>
-        <width>838</width>
-        <height>1015</height>
-      </rect>
-    </property>
-    <property name="windowTitle">
-      <string>PDOK Services Plugin</string>
-    </property>
-    <layout class="QVBoxLayout" name="verticalLayout">
-      <item>
-        <widget class="QTabWidget" name="tabWidget">
-          <property name="sizePolicy">
-            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
+ <class>PdokServicesPlugin</class>
```

Comment:
voor zover ik heb gezien is dit een styling fix, niet heel intensief doorheen gekeken.